src/MacAppBundle/CMakeLists.txt

```diff
@@ -84,13 +109,25 @@ install(CODE "execute_process(COMMAND chmod -R a+w ${CMAKE_INSTALL_LIBDIR})")
 
 get_filename_component(APP_PATH ${CMAKE_INSTALL_PREFIX} PATH)
 
+execute_process(
+       COMMAND find /usr/local/Cellar/icu4c -name pkgconfig
+       RESULT_VARIABLE CMD_ERROR
+       OUTPUT_VARIABLE CONFIG_ICU)
+set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:${CONFIG_ICU}")
+find_package(PkgConfig)
+pkg_check_modules(ICU icu-uc)
+
+execute_process(
+       COMMAND find /usr/local/Cellar/nglib -name MacOS
+       OUTPUT_VARIABLE CONFIG_NGLIB)
+
 install(CODE 
     "message(STATUS \"Making bundle relocatable...\")
     # The top-level CMakeLists.txt should prevent multiple package manager
     # prefixes from being set, so the lib path will resolve correctly...
     execute_process(
-        COMMAND ${PYTHON_EXECUTABLE}
+        COMMAND python2.7
```

Comment:
why hardcode python2.7 here ?

Reply:
The script is broken in python 3.8. It needs to be updated, and it is part of my TODO. If we call it currently it will fail and still need some homework to be ported to 3.8